.gitignore

```diff
@@ -46,3 +45,4 @@ __pycache__
 /tests/unit/bintest/
 /tests/unit/objtest/
 /tests/ystests
+build
```

Comment:
Should this not be `/build`? This would ignore every file or directory called `build` in the entire tree, which might unpleasantly surprise someone.